### ISSUE TITLE
Windows: Error in loadNamespace(name) : there is no package called 'BiocInstaller'

### ISSUE THREAD

**Issue by HenrikBengtsson** [r-hub/rhub#102]
The following error is reproducible and occurs on other packages too:
```r
> rhub::check_on_windows()
─  Building package
─  Uploading package
─  Preparing build, see status at
   http://builder.r-hub.io/status/listenv_0.6.0-9000.tar.gz-99ddbef0be2f4ef396f1edd41d592e5b
─  Build started
─  Creating new user
─  Downloading and unpacking package file
─  Querying package dependencies
─  Installing package dependencies
─  Running R CMD check
   setting _R_CHECK_FORCE_SUGGESTS_ to false
   Error in loadNamespace(name) : there is no package called 'BiocInstaller'
   Calls: options ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
   Execution halted
   
   
   +R-HUB-R-HUB-R-HUB Done.
   
   
   VERBOSE: Saving artifacts
   cp : Cannot find path 'C:\Users\USERxurLBXJFkc\listenv.Rcheck' because it does 
   not exist.
   At C:\Users\rhub\Documents\workspace\listenv_0.6.0-9000.tar.gz-99ddbef0be2f4ef3
   96f1edd41d592e5b\run.ps1:189 char:1
   + cp -Recurse ( $homefull + "\" + $pkgname + ".Rcheck" ) $jobid | Out-Null
   + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       + CategoryInfo          : ObjectNotFound: (C:\Users\USERxurLBXJFkc\listenv 
      .Rcheck:String) [Copy-Item], ItemNotFoundException
       + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyI 
      temCommand
    
   VERBOSE: Cleaning up, deleting files and user
   >>>>>============== Cleaning up files and user
   SSH: Connecting from host [win-builder-1]
   SSH: Connecting with configuration [files] ...
   SSH: Disconnecting configuration [files] ...
   SSH: Transferred 0 file(s)
   Build step 'Send files or execute commands over SSH' changed build result to SUCCESS
   Pinging https://builder.r-hub.io/build/SUCCESS/listenv_0.6.0-9000.tar.gz-99ddbef0be2f4ef396f1edd41d592e5b/2018-01-04T01:12:59Z
   {"status":"ok"}
   Finished: SUCCESS
```

**Comment by gaborcsardi** [comment]
I think this does not happen any more. It was because of the BioC transition to a new version. We do need to think about getting rid of these hiccups, though.

**Comment by privefl** [comment]
I get the same error while running `rhub::check_with_sanitizers(devtools::build())` (see [the build](https://builder.r-hub.io/status/bigsnpr_0.3.1.tar.gz-01bd7cfffe684d65a3216800663d7826)).

Note that the package doesn't need any Bioconductor package.

**Comment by MikeBadescu** [comment]
The error still appears: https://builder.r-hub.io/status/shinyXT_0.1.0.tar.gz-70e48876c8886ca4dc5bd0315ec128b7

**Comment by michaelspence** [comment]
I also get this error here:
https://builder.r-hub.io/status/LeMaRns_0.0.0.9000.tar.gz-6f7c45e49cb365638e4962b52c44405b

**Comment by maelle** [comment]
Cf open issue https://github.com/r-hub/rhub/issues/279

**Comment by maelle** [comment]
@michaelspence @kylebaron @andrewGhazi the problem has been solved cf #279
